src/Cthulhu.jl

```diff
@@ -294,10 +303,10 @@ function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::Method
             # This was inferred to a pure constant - we have no code to show,
             # but make something up that looks plausible.
             if display_CI
-                println(iostream)
-                println(iostream, "│ ─ $(string(Callsite(-1, MICallInfo(mi, interp.opt[mi].rettype), :invoke)))")
-                println(iostream, "│  return ", Const(interp.opt[mi].rettype_const))
-                println(iostream)
+                println(term.out_stream::IO)
```

Comment:
Why did you change all these?

Reply:
Hmm, maybe a rebase error, I changed them back though, thanks